modules/install-central-operator-tech-preview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * installing/install-ocp-operator.adoc
+:_module-type: PROCEDURE
+[id="install-central-operator-tech-preview_{context}"]
+= Installing Central with an external database using the Operator method
```

Comment:
Installing Central with an external database using the {product-title-short} Operator